org.eclipse.xtext.testing/META-INF/MANIFEST.MF

```diff
@@ -28,9 +28,16 @@ Require-Bundle: org.eclipse.xtext;visibility:=reexport,
  org.eclipse.xtend.lib
 Import-Package: org.apache.log4j;version="1.2.24",
  org.apache.log4j.spi;version="1.2.24",
- org.junit.jupiter.api;version="[5.0.0,6.0.0)";resolution:=optional,
- org.junit.jupiter.api.extension;version="[5.0.0,6.0.0)";resolution:=optional,
- org.junit.platform.commons.support;version="[1.0.0,2.0.0)";resolution:=optional
+ org.junit.jupiter.api;version="[5.1.0,6.0.0)",
```

Comment:
IIRC, this might not work during the build? Or it will fail if you forget to add that to the TP. Do you want me to try that? However, that dependency is needed, again, IIRC.